test/stache-test.js

```diff
@@ -6580,6 +6594,44 @@ function makeTest(name, doc, mutation) {
 		equal(warn1(), 0);
 		equal(warn2(), 0);
 	});
+
+	test('#each with call expression and plain array should not use utils.getItemsFragContent', function(){
```

Comment:
The other tests for arrays+call expressions+hash expressions test that this works, I don't think it's necessary to test this implementation detail.